addons/sale_timesheet/tests/test_project.py

```diff
@@ -4,6 +4,7 @@
 from odoo import Command
 from .common import TestCommonSaleTimesheet
 from odoo.tests import tagged
+from odoo.tests.common import Form
```

Comment:
Form is normally available in `odoo.tests`. 🙂 
```suggestion
from odoo.tests import tagged, Form
```

Reply:
thank you